src/components/wormhole.js

```diff
@@ -15,18 +15,32 @@ export class Wormhole {
 
     transport.passengers = freeze(passengers)
     const keys = Object.keys(this.transports)
-    if (keys.indexOf(to) !== -1) {
-      this.transports[to] = transport
+    if (keys.indexOf(to) === -1) {
+      Vue.set(this.transports, to, [])
+    }
+
+    const currentIndex = this.getTransportIndex(transport)
+    if (currentIndex === -1) {
+      this.transports[to].push(transport)
```

Comment:
I think we should maybe use `.shift` instead, so that for Targets with `multiple=false`, the latest source is displayed at all times.

Reply:
I definitely agree that I'd expect the last one to be the one to appear if multiple=false. But if we use shift the portals will appear in reverse order if multiple=true.

Maybe we could change the last line of ownTransports (in portal-target) to this instead:
   return transports.length === 0 ? [] : [transports[transports.length - 1]]

For an additional feature idea:
Maybe it would be nice to add a :order prop to portals so users can decide in which order the portals should appear when multiple is true.
(Although I haven't actually thought of a use case yet, since I'm currently only planning to use multiple for moving all my modals to the end of the DOM.)

Reply:
Yeah, goping for the last element is even better I think. Will review in more depth this weekend.

Nice work, many thanks! :)